src/components/form/Form.tsx

```diff
@@ -129,16 +127,16 @@ const Form = () => {
 
         <FormButton
           type={"submit"}
-          color={Color.cream}
-          backgroundColor={Color.mediumGreen}
+          color={theme.colors.cream}
+          backgroundColor={theme.colors.mediumGreen}
           handleClick={submitFormData}
         >
           zapisz
         </FormButton>
       </StyledForm>
 
       <h4>from local storage</h4>
-      <p style={{ color: Color.cream }}>
+      <p style={{ color: theme.colors.cream }}>
```

Comment:
please use `styled components` instead of inline styles for consistency and readability

Reply:
I'll remove it entirely. It was there to check if data was retrieved and is not really a part of design.